### PR TITLE
Feat : product listing

### PR DESCRIPTION
**Pull request by yogesh-**
Added product listing page containing:
- product grid
- Filters
   - Price
   - Category
   - Ratings

image

image

### CODE REVIEW

File: product_listing.html [hunk 1]
```diff
@@ -0,0 +1,348 @@
...
+
+                <div class="nav-items">
+                    <ul class="flex-row right-icons">
+                        <li><button class="btn">
+                                <p class="h4">Login</p>
+                        </li>
+
+
+
+
```
Comment (akash-1318) [inline product_listing.html]: remove these extra line spacing

File: product_listing.css [hunk 2]
```diff
@@ -0,0 +1,200 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  --primary-color: #03045e;
+  --text-color: white;
+  --black-color: black;
+  --hover-btn-color: #caf0f8;
```
Comment (syed-shaida-hussain) [inline product_listing.css:8]: You can declare color variables inside :root

File: product_listing.css [hunk 3]
```diff
@@ -0,0 +1,200 @@
...
+}
+
+#page footer {
+  grid-area: footer;
+}
+#page sidebar {
+  background-color: var(--text-color);
+  grid-area: filter;
+  position: -webkit-sticky;
+  position: sticky;
```
Comment (syed-shaida-hussain) [inline product_listing.css:40]: You are using position property twice

File: product_listing.css [hunk 4]
```diff
@@ -0,0 +1,200 @@
...
+  height: 5rem;
+}
+
+.header-row .nav-items button {
+  background-color: var(--text-color);
+  color: var(--black-color);
+}
+
+.header-row .nav-items .number-red {
+  background-color: tomato;
```
Comment (syed-shaida-hussain) [inline product_listing.css:71]: Try to use variable for color here as well as you are using variables everywhere.

File: product_listing.html [hunk 5]
```diff
@@ -0,0 +1,348 @@
...
+                        <li>
+                            <div class="badge">
+                                <span class="material-icons-outlined">
+                                    shopping_cart
+                                </span>
+                                <div class="number-red">12</div>
+                            </div>
+
+                        </li>
+
```
Comment (syed-shaida-hussain) [inline product_listing.html]: Remove unwanted lines.

**Review by syed-shaida-hussain — CHANGES_REQUESTED** [review]

File: product_listing.css (hunk 2 above)
Comment (yogesh-) [reply product_listing.css:8]: okay, done

**Pull request merged**